mne/minimum_norm/inverse.py

```diff
@@ -1277,6 +1285,149 @@ def apply_inverse_epochs(epochs, inverse_operator, lambda2, method="dSPM",
     return stcs
 
 
+def _apply_inverse_tfr_gen(average_tfr, inverse_operator, lambda2,
+                           method, label, pick_ori, prepared,
+                           method_params, use_cps):
+    for freq_idx in range(average_tfr.freqs.size):
+        evoked = EvokedArray(average_tfr.data[:, freq_idx, :],
+                             average_tfr.info, tmin=average_tfr.tmin)
+        stc = _apply_inverse(
+            evoked, inverse_operator, lambda2, method=method,
+            pick_ori=pick_ori, prepared=prepared, label=label,
+            method_params=method_params, return_residual=False,
+            use_cps=use_cps)
+        yield stc
+
+
+@verbose
+def apply_inverse_tfr(average_tfr, inverse_operator, lambda2,
```

Comment:
I don't think this makes sense. I don't really see a use case where you would average the complex valued EpochsTFR. You would look all the interest for TFR to find induced activity. I would not expose such a method.